docs/CODE-QUALITY.md

```diff
@@ -0,0 +1,24 @@
+| Unit Tests                                                             | Automated tests to ensure functionality is valid and helps with regression testing.                                                                                                                                                                                                                                                                                 |
+| Functional Testing                                                     | Test Plans are created for each sprint to ensure new enhancements, features and bugs are manually tested.                                                                                                                                                                                                                                                           |
+| Integration Testing                                                    | We have limited integration testing. The API automates an in-memory database to test the Data Access Layer (DAL).                                                                                                                                                                                                                                                   |
+| Automated Functioning Testing                                          | There is currently no automated functional testing.                                                                                                                                                                                                                                                                                                                 |
+| ZAP                                                                    | 3rd party tool to review code security quality. **Not presently hooked up to DevOps pipeline** pipeline.                                                                                                                                                                                                                                                            |
+| [Snyk](https://app.snyk.io/org/fosol/)                                 | Tool to review code dependencies and vulnerabilities.                                                                                                                                                                                                                                                                                                               |
```

Comment:
_Not presently hooked up to DevOps pipeline_

Also it would be great if we could create an "org" account and stop configuring services under our own personal GitHub - tech debt but food for thought